pkg/registry/core/pod/strategy.go

```diff
@@ -381,14 +381,13 @@ func streamParams(params url.Values, opts runtime.Object) error {
 			params.Add(api.ExecTTYParam, "1")
 		}
 	case *api.PodPortForwardOptions:
-		if len(opts.Ports) == 0 {
```

Comment:
Yes it is ./pkg/kubelet/server/portforward/websocket.go:65. It fails when it is a websocket request and allows it for spdy. I am going to make a quick fix on the message, it isn't proper english.